### DISCUSSION TITLE
Single project solution like MAUI

### DISCUSSION THREAD

**Discussion by pwhe23** [unoplatform/uno#4886] [Q&A] [answered]
It's been a while since the last time I took Uno for a test drive. I have read that MAUI in .Net 6 plans to support single-project solutions rather than a separate project per platform. Are there any plans for Uno to support something similar? I'm assuming this might be a challenge with the current architecture but am just curious since I couldn't find a discussion of that anywhere.

**Comment by francoistanguay** [comment, marked as answer]
Yes, everything .net 6 is bringing to the table, Uno will be able to benefit from, namely Single-project solutions. 

In a sense, it's already working but need Visual Studio upcoming support for it to be useable. Stay tuned!

**Reply by listepo** [reply]
@francoistanguay any news?

**Reply by francoistanguay** [reply]
What's the question? We've had net 6 support for several months now https://platform.uno/blog/uno-platform-3-9-net-6-mobile-vs-2022-winui-plus-xaml-trimming-for-50-wasm-aot-size-reduction/

Our next 4.1 release, coming roughly a week from now will align with the latest Preview 13 update of net6/Mobile.

**Reply by listepo** [reply]
@francoistanguay, sorry, any news for other platforms?

**Reply by francoistanguay** [reply]
Yeah, sadly there's nothing we can do beyond the platforms currently supported through TFMs in net6.

**Reply by listepo** [reply]
@francoistanguay understood, thanks

**Comment by HavenDV** [comment]
At the moment, you can try the single project solution within a separate repo - https://github.com/HavenDV/Uno.Sdk. This is a workload similar to tizen and this delivers`net7.0-webassembly;net7.0-gtk;net7.0-linux` to distinguish WebAssembly/Skia.Gtk/Skia.Linux.Framebuffer respectively. Skia.Wpf is implemented within `net7.0-windows` (this is an implicit indication for `net7.0-windows7`, which is different from the WinUI TargetFramework - `net7.0-windows10.x.x.x`).
Check the documentation for more details.
I would be glad to receive any feedback
